.build/external-documentation/rhoas-api-overview.quickstart.yml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: docs-rhoas-api-overview
   annotations:
     order: 5
+  labels:
+    service-category/application-services: true
+    product/rhosak: true
+    product/rhoam: true
```

Comment:
Same comment as https://github.com/redhat-developer/app-services-guides/pull/617/files#r1095633832. Don't this applies to RHOAM.

Reply:
I think this doc applies to RHOSR (it's listed on the RHOSR docs splash page) so we should probably add product/rhosr to the list? @smccarthy-ie or @jbyrne-redhat can confirm.

Reply:
You're right, @bredamc - it also applies to RHOSR. Thanks for catching.

Reply:
+1

Reply:
replaced RHOAM with product/rhosr: "true"